adot/utils/soak/soak.py

```diff
@@ -50,8 +50,12 @@ def enableLambdaInsight(function_name, architecture):
 
 
 def parse_args():
-    # default setting
-    _soaking_time, _emitter_interval, _cpu_threshold, _memory_threshold = 10000, 5, 60, 45
+    _soaking_time = 10000
+    _emitter_interval = 5
+    _cpu_threshold = 100 # total cpu time in ms. This value was obtained
```

Comment:
I'm not sure this will have the desired effect.  See [here](https://github.com/aws-observability/aws-otel-lambda/blob/main/.github/workflows/soaking.yml#L87-L95).

Reply:
:(